packages/helpers/packages/TemplateHelpers/index.js

```diff
@@ -80,4 +80,9 @@ export const makeShadow = (
   spread,
   color,
   inset = false,
-) => `${decideMeasurement(hOffset)} ${decideMeasurement(vOffset)} ${decideMeasurement(blur)} ${decideMeasurement(spread)} ${color}${inset ? ' inset' : ''}`;
+) =>
+  `${decideMeasurement(hOffset)} ${decideMeasurement(
+    vOffset,
+  )} ${decideMeasurement(blur)} ${decideMeasurement(spread)} ${color}${
+    inset ? ' inset' : ''
```

Comment:
inset && ' inset' is better?

Reply:
actually there is no such difference and this does not make big sense